test/fixtures/flow/type-annotations/136/actual.js

```diff
@@ -0,0 +1,4 @@
+type A = {
+	p: {},
+	...{},
+};
```

Comment:
This should parse. How are you testing? Gabe's comment is that parsePrimaryType is actually *too* restrictive. We also parse non-primary types in this position, like unions. E.g., `{...A|B}`.